source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Configuration/SharedConfiguration.cs

```diff
@@ -65,6 +73,14 @@ internal static class SharedConfiguration
     {
         internal static void ConfigureServices(IServiceCollection serviceCollection)
         {
+            var serviceBusConnectionString = EnvironmentHelper.GetEnv(EnvironmentSettingNames.DataHubSenderConnectionString);
+            var dataAvailableQueue = EnvironmentHelper.GetEnv(EnvironmentSettingNames.MessageHubDataAvailableQueue);
+            var domainReplyQueue = EnvironmentHelper.GetEnv(EnvironmentSettingNames.MessageHubReplyQueue);
```

Comment:
`domainReplyQueue` seems a bit misleading, how about just using

```suggestion
            var messageHubReplyQueue = EnvironmentHelper.GetEnv(EnvironmentSettingNames.MessageHubReplyQueue);
```

Reply:
Done